src/server/templates/target.js

```diff
@@ -0,0 +1,28 @@
+import favicon from './favicon';
+
+
+export default ({title, markup, page, props, script}) => `
+	  <!DOCTYPE html>
+	  <html>
+		<head>
+		  <title>${title ? `'${title} – BookBrainz'` :
+		     'BookBrainz – The Open Book Database'}</title>
+		  <link rel="stylesheet" href='/stylesheets/style.css' />
+		  <link rel="stylesheet" href='/stylesheets/react-virtualized.css' />
+		  <link rel="stylesheet"
+		    href='/stylesheets/react-virtualized-select.css' />
+		  <meta name='viewport' content='//react-virtualized.css' />
+		  ${favicon}
+		</head>
+
+		<body>
+		  <div id="target">${markup}</div>
+		  <script src='/js/bundle.js' />
+		  ${page ?
```

Comment:
It'd be clearer to use a logical `&&` here instead of a ternary expression since the falsy condition is empty. Do the same for the other ternary expression below.